### ISSUE TITLE
Installation error with 1.2.4

### ISSUE THREAD

**Issue by jingsliu** [blei-lab/edward#568]
Hi I got the following error importing the package after successful installation (pip --user on Linux):

>>> import edward as ed
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "xxx/.local/lib/python2.7/site-packages/edward/__init__.py", line 6, in <module>
    from edward import inferences
  File "xxx/.local/lib/python2.7/site-packages/edward/inferences/__init__.py", line 11, in <module>
    from edward.inferences.laplace import *
  File "xxx/.local/lib/python2.7/site-packages/edward/inferences/laplace.py", line 9, in <module>
    from edward.models import \
ImportError: cannot import name MultivariateNormalCholesky

Could you please help check? 
Also it'll be great if you can provide url of the binaries of previous versions. Thanks!

**Comment by dustinvtran** [comment]
Can you provide your TensorFlow version? Most likely this means you need to update TensorFlow. Also see http://edwardlib.org/troubleshooting.

**Comment by jingsliu** [comment]
It's version 1.0.0 so I thought it was fine.

**Comment by dustinvtran** [comment]
Can you try the following line?
```python
from tensorflow.contrib.distributions import MultivariateNormalCholesky
```
[This should be supported in r1.0](https://www.tensorflow.org/api_docs/python/tf/contrib/distributions/MultivariateNormalCholesky).

If you're using a master branch and not a stable release of TensorFlow, this isn't supported. (TensorFlow's distribution API changed, which we haven't officially supported yet (https://github.com/blei-lab/edward/pull/452).)

**Comment by jingsliu** [comment]
Thanks and that's it.
Unfortunately I'm working with the tensorflow installed by the system administrator... Any idea if an older version of tensorflow might work, or how soon would you expect this being supported?

**Comment by dustinvtran** [comment]
All TensorFlow version releases after `1.0.0a0` are supported. 

> how soon would you expect this being supported?

We don't plan to support the newest API changes in TensorFlow's master branch until it makes it into a stable release.

Hope that helps. Closing issue; also see #452 for more details.

**Comment by bhomass** [comment]
I installed edward using conda and got version and got version 1.2.4. It automatically included tensorflow version 1.5.0. I am getting this ImportError: cannot import name 'MultivariateNormalCholesky' still, even for this version. What should I do to fix?

**Comment by dustinvtran** [comment]
Edward 1.2.4 is definitely out of date; I also don't recall what TF version it's compatible with. Maybe update to latest Edward?
